merkle/simple_tree.go

```diff
@@ -54,28 +54,25 @@ func SimpleHashFromHashes(hashes [][]byte) []byte {
 	}
 }
 
-// Convenience for SimpleHashFromHashes.
-func SimpleHashFromBinaries(items []interface{}) []byte {
-	hashes := make([][]byte, len(items))
-	for i, item := range items {
-		hashes[i] = SimpleHashFromBinary(item)
+// NOTE: Do not implement this, use SimpleHashFromByteslices instead.
```

Comment:
Do not implement what?

Reply:
SimpleHashFromBytesers.  Better to just use SimpleHashFromByteslices.